src/App/Application.cpp

```diff
@@ -2521,7 +2521,7 @@ void processProgramOptions(const variables_map& vm, std::map<std::string,std::st
         if (pos != mConfig.end()) {
             str << pos->second;
         }
-        str << std::endl;
+        str << '\n';

```

Comment:
Why should we use `\n` and not `std::endl`?

Reply:
I agree, and `using namespace std;` should be removed so we keep `std::`

Reply:
@bofdahof which unwanted behaviour?

Reply:
I've missed the flush part, thanks for pointing that out.
Also it's worth pointing out that `std::endl` doesn't take care of platform-specific line endings such as `\r\n` on windows: https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rio-endl